packages/core/src/browser/theming.ts

```diff
@@ -196,6 +196,7 @@ export class BuiltinThemeProvider {
     // Webpack converts these `require` in some Javascript object that wraps the `.css` files
     static readonly darkCss = require('../../src/browser/style/variables-dark.useable.css');
     static readonly lightCss = require('../../src/browser/style/variables-bright.useable.css');
+    static readonly materialColors = require('../../src/browser/style/materialcolors.css');
```

Comment:
I think it should go here: https://github.com/theia-ide/theia/blob/ak/plugins_reconnect/packages/core/src/browser/frontend-application-module.ts#L17 and be used always

why -> https://github.com/theia-ide/theia/pull/6155/files#r323145830

Reply:
Thanks. Done.

Reply:
@olexii4 please move the import after this line: https://github.com/theia-ide/theia/blob/ak/plugins_reconnect/packages/core/src/browser/frontend-application-module.ts#L17 to make sure that it is loaded exactly with the same timing, thank you!